docs/src/consensus/block.md

```diff
@@ -24,7 +24,7 @@ Besides the kernel, blocks also contain proofs. The block proof is a STARK proof
 
 ## Validity
 
-**Note:** this section describes the validity rules for blocks at some future point when we have succinctness, not the current validity rules (although there is a significant overlap).
+**Note:** this section describes two distinct things. First is the validity rules for blocks on the presently deployed network. Second is the validity rules as they are intended to be on some future fork, for instance the one that introduces succinctness.
```

Comment:
I don't like the use of the work "network" in this context, as it sounds like a reset of the monetary distribution which we don't intend. Can the word "network" be replaced by "consensus rules"?